src/components/course/enrollment/tests/EnrollAction.test.jsx

```diff
@@ -217,5 +225,12 @@ describe('scenarios user not yet enrolled, but eligible to enroll', () => {
     expect(screen.getByText(enrollLabelText).closest('a')).not.toBeInTheDocument();
     const regex = new RegExp().compile(createUseVoucherText(1));
     expect(screen.getByText(regex)).toBeInTheDocument();
+
+    const PAYMENT_TEXT = 'Enroll in course';
+    // also check url is rendered in the modal correctly
```

Comment:
Same as above, I'd prefer a separate test for this.

Reply:
will do, sounds like a good idea